GameData/RP-1/Programs/Programs.cfg

```diff
@@ -287,11 +288,109 @@ RP0_PROGRAM
 		Fast = 600
 	}
 }
+RP0_PROGRAM
+{
+	name = EarlyEOS
+	title = Early Earth Observation Satellites
+	description = Now that you have proven the ability to send satellites into orbit, the next step is to send them into more useful orbits with beneficial payloads. The goal of this program is to put satellites with different payload requirements into specific and more challenging orbits. 
+	requirementsPrettyText = Complete Early Satellites program
+	objectivesPrettyText = Launch satellites to analyze weather, measure solar wind and cosmic rays, and determine the extent of the Magnetosphere.
+	nominalDurationYears = 3
+	baseFunding = 420000
+	repDeltaOnCompletePerYearEarly = 130
+	repPenaltyPerYearLate = 130
+	slots = 1
+
+	REQUIREMENTS
+	{
+		ANY
+		{
+			complete_program = EarlySatellites
+			complete_program = EarlySatellites-Heavy
+		}
+  		ANY
+    		{
+      			active_program = CrewedOrbit
+	 		active_program = CrewedOrbitEarly
+    			complete_program = CrewedOrbit
+       			complete_program = CrewedOrbitEarly
+	  	}
+	}
+
+	OBJECTIVES
+	{
+		complete_contract = EOSCosmicRay1
+		complete_contract = EOSCherenkov
+		complete_contract = EOSMagScan1
+		complete_contract = EOSIRRad1
+  		complete_contract = EOSCorona
+	}
+	
+	OPTIONALS
+	{
+		EOSIRRad1Rep = true
+   		EOSGrav1 = true
+		EOSCoronaRep = true
+	}
+
+	CONFIDENCECOSTS
+	{
+		Normal = 300
+		Fast = 600
+	}
+}
+
+RP0_PROGRAM
+{
+	name = EarlyCommApp
+	title = Early Commercial Satellites
+	description = Although it was through government programs that the earliest satellites were launched, commercial use of them was always envisioned as well.  Through this program, the earliest iterations of commercial-use satelites will be launched.  With wildly varying needs, each satellite is dedicated to a specific requirement, typically far more focused than the earlier broad scientific satellites. 
+	requirementsPrettyText = Complete Early Satellites program
+	objectivesPrettyText = Launch the first commercial-use satellites and build a satellite communication network. 
+	nominalDurationYears = 2
```

Comment:
With the two networks now this is at least 3 years I think. It's possible both should be 4 but I'm unsure, I'll let Max speak more authoritatively :D

Reply:
Agreed on 3 years.